src/test/groovy/com/onesignal/androidsdk/MainTest.groovy

```diff
@@ -1028,16 +1065,17 @@ class MainTest extends Specification {
     //       One to this test may cause duplicated or missing classes with either of these
     //       Might need to follow the AndroidX migration guide to fix and re-add support library
     def 'Find min-support for Firebase and GMS - build'() {
-        GradleTestTemplate.buildArgumentSets[GRADLE_OLDEST_VERSION] = [['transformClassesAndResourcesWithProguardForDebug']]
+        GradleTestTemplate.buildArgumentSets[GRADLE_LATEST_VERSION] = [['build']]
         when:
         // Keep as '+' for latest when checking in to this fails when Google changes requirements
```

Comment:
What is this comment in reference to now since you deleted the `gms_version = '+'`

Reply:
Good catch, I'll clean this up